README.md

```diff
@@ -37,6 +37,24 @@ jobs:
 +        args: analyse src/ --level=5
```

Comment:
Good catch 👍

Reply:
This was already here - it just looks like an addition on this PR because of the `+` 😄

Reply:
ah 😃